tests/test_reader_modules.py

```diff
@@ -3,6 +3,12 @@
 """
 import pytest
 import GCRCatalogs
+import os
+
+if GCRCatalogs.register._ROOT_DIR is None:
+    # Adust so test can be run anywhere
+    GCRCatalogs.register.set_root_dir(os.getenv('HOME'))
+    GCRCatalogs.register._ROOT_DIR = str(os.getenv('HOME'))
```

Comment:
I think eventually we agree that `GCRCatalogs` and its modules should be importable even if ROOT_DIR is not set?
```suggestion
```